Taggers/python/flashggHHWWggTag_cfi.py

```diff
@@ -54,27 +53,42 @@
                                     DeltaRTrkElec = cms.double(.4),
                                     TransverseImpactParam = cms.double(0.02),
                                     LongitudinalImpactParam = cms.double(0.2),
-                                    deltaRPhoElectronThreshold = cms.double(0.4), # was 1 
-                                    deltaMassElectronZThreshold = cms.double(10.),                                  
+                                    deltaRPhoElectronThreshold = cms.double(0.4), # was 1
+                                    deltaMassElectronZThreshold = cms.double(10.),
+                                    deltaMassElectronZ_FL_Threshold = cms.double(5.),
```

Comment:
Instead of having a separate deltaMassElectronZ threshold for the fully leptonic case, should just set deltaMassElectronZThreshold value to desired value in HHWWggCustomize (added in next commit)